README.md

```diff
@@ -1,58 +1,114 @@
 
-# Introduction
+[Wayfire]: https://wayfire.org
 
-[![Join the chat at https://gitter.im/Wayfire-WM/Lobby](https://badges.gitter.im/Wayfire-WM/Lobby.svg)](https://gitter.im/Wayfire-WM/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) or join #wayfire at freenode.net
+![Version](https://img.shields.io/badge/version-alpha-important)
+[![IRC](https://img.shields.io/badge/irc-%23wayfire-informational)](https://webchat.freenode.net/#wayfire)
 
-Wayfire is a wayland compositor based on wlroots. It aims to create a customizable, extendable and lightweight environment without sacrificing its appearance. If you want to gain a better impression at what it can do, see the demo videos on youtube: [link](https://www.youtube.com/playlist?list=PLb7YRKEhWEBUIoT-a29UoJW9mhfzjpNle)
+###### [Get started] | [Manual] | [Configuration]
 
-# Build and install
+[Get started]: https://github.com/WayfireWM/wayfire/wiki/Tutorial
+[Manual]: https://github.com/WayfireWM/wayfire/wiki/Manual
+[Configuration]: https://github.com/WayfireWM/wayfire/wiki/Configuration
 
-To build wayfire, you'll need `glm`, `wf-config`(built as a submodule by default) and `wlroots` + its build dependencies. When ready, simply clone this repo, compile and install:
+Wayfire is a 3D [Wayland] compositor, inspired by [Compiz] and based on [wlroots].
 
-```
-git clone https://github.com/WayfireWM/wayfire && cd wayfire
-meson build --prefix=/usr --buildtype=release
-ninja -C build && sudo ninja -C build install
+It aims to create a customizable, extendable and lightweight environment without sacrificing its appearance.
+
+[![Wayfire demos](https://img.youtube.com/vi_webp/2PtNzxDsxYM/maxresdefault.webp)](https://youtube.com/playlist?list=PLb7YRKEhWEBUIoT-a29UoJW9mhfzjpNle "YouTube – Wayfire demos")
+[![YouTube Play Button](https://www.iconfinder.com/icons/317714/download/png/16)](https://youtube.com/playlist?list=PLb7YRKEhWEBUIoT-a29UoJW9mhfzjpNle) · [Wayfire demos](https://youtube.com/playlist?list=PLb7YRKEhWEBUIoT-a29UoJW9mhfzjpNle)
+
+[Wayland]: https://wayland.freedesktop.org
+[wlroots]: https://github.com/swaywm/wlroots
+[Compiz]: https://launchpad.net/compiz
+
+## Dependencies
+
+- [Cairo](https://cairographics.org)
+- [FreeType](https://freetype.org)
+- [GLM](https://glm.g-truc.net)
+- [libdrm](https://dri.freedesktop.org/wiki/DRM/)
+- [libevdev](https://freedesktop.org/wiki/Software/libevdev/)
+- [libGL](https://mesa3d.org)
+- [libinput](https://freedesktop.org/wiki/Software/libinput/)
+- [libjpeg](https://libjpeg-turbo.org)
+- [libpng](http://libpng.org/pub/png/libpng.html)
+- [libxkbcommon](https://xkbcommon.org)
+- [Pixman](https://pixman.org)
+- [pkg-config](https://freedesktop.org/wiki/Software/pkg-config/)
+- [Wayland](https://wayland.freedesktop.org)
+- [wayland-protocols](https://gitlab.freedesktop.org/wayland/wayland-protocols)
+- [wf-config](https://github.com/WayfireWM/wf-config)
+- [wlroots](https://github.com/swaywm/wlroots)
+
+## Installation
+
```

Comment:
Can you also add the comments about building `wf-config` and `wlroots` as submodules?

Reply:
And also the warning that they will be automatically built as submodules if not present on the system already.